cypress/integration/test1.spec.js

```diff
@@ -0,0 +1,14 @@
+describe('ut2019', function() {
+    const languagesHome = ['https://ut2019.voog.com/en', 'https://ut2019.voog.com/et'];
+    const languagesContact = ['https://ut2019.voog.com/contact', 'https://ut2019.voog.com/contact-1']
+    const contact = ['a[href="/contact"]', 'a[href="/contact-1"]']
+    const home = ['a[href="/en"]', 'a[href="/et"]']
```

Comment:
Your solution for testing different languages is OK, but a better solution has been implemented here:
https://github.com/ut2019-testing/ut2019/blob/bcae98d57e7224912a3a8e9a50ba3bc28002ef75/cypress/integration/navigateHomeAndContactsPages.spec.js#L3

Currently the three lists must always have the elements in the same order. With two elements it is manageable, but with 20 it would be a hassle.